src/org/gk/database/util/ChEBIAttributeAutoFiller.java

```diff
@@ -16,7 +16,7 @@
 import org.gk.persistence.XMLFileAdaptor;
 import org.junit.Test;
 
-import uk.ac.ebi.demo.ols.soap.Query;
+import uk.ac.ebi.ols.soap.Query;
```

Comment:
What version of ols-client are you suggesting? My current version used for the curator tool doesn't have this package.

Reply:
The version I have found while compiling the dependencies for assembling a POM file is:
```
<dependency>
    <groupId>uk.ac.ebi.ols</groupId>
    <artifactId>ols-client</artifactId>
    <version>1.18</version>
</dependency>
```
available here: https://mvnrepository.com/artifact/uk.ac.ebi.ols/ols-client/1.18